script.js

```diff
@@ -0,0 +1,68 @@
+/* gameBoard array includes all list tags
+   used in event listen array to only listen for clicks on game board*/
+var gameBoard = document.getElementsByTagName("li");
```

Comment:
While this is technically correct, I'd advocate for using a className to select which elements are part of the board. The reason being that you may later add other `li`s, which are unrelated to the board (e.g. a list of rules), and this could have unintentional consequences.

If you go this route, you'll want to make sure you add classes when marking a square, not over-writing them.